### PR TITLE
update goldenforge status

### PR DESCRIPTION
**Pull request by manugame**
goldenforge is no longer Inactive

### CODE REVIEW

**Comment by manugame** [comment]
i wasn't even aware that goldenforge was in this list x)

**Comment by TonimatasDEV** [comment]
Don't delete Inactive Developement, only move GoldenForge to up

File: java/MODS.md [hunk 1]
```diff
@@ -27,12 +27,11 @@ If you're looking for a way to use both **plugins and mods**, checkout [this lis
...
 
-# ❌ Inactive Development
 ### [⭐ GoldenForge](https://github.com/GoldenForge/GoldenForge)
 - **Alternative**: Forge
-- **Version:** 1.16.5
+- **Version:** 1.19.2
 - **Author:** manugame
 - **Fork: Forge** --> GoldenForge
-- **Description:** Fork of Forge 1.16.5 which implement some Paper patches
+- **Description:** Fork of Forge 1.19.2 which implement some Paper patches
```
Comment (UeberallGebannt) [inline java/MODS.md:35]: Please move it up instead of deleting the header :)
Also, please add all covered versions, even if inactive (you can add a note).

**Review by UeberallGebannt — CHANGES_REQUESTED** [review]

**Comment by UeberallGebannt** [comment]
@manugame Thank you for updating this list, for some reason github decided not to inform me about your PR, which is why I didn't respond earlier 😅

**Pull request closed without merging**